Misc/NEWS.d/next/Library/2022-01-21-18-19-45.bpo-41906.YBaquj.rst

```diff
@@ -0,0 +1,2 @@
+Support passing filter instances to the ``filters`` values of ``handlers``
+and ``loggers`` in :func:`logging.config.dictConfig`.
```

Comment:
change this to

Support passing filter instances in the ``filters`` values of ``handlers`` and ``loggers`` in the dictionary passed to :func:`logging.config.dictConfig`.